build/commands/lib/pylint.js

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+const path = require('path')
+const fs = require('fs')
+const config = require('../lib/config')
+const util = require('../lib/util')
+
+const getDefaultOptions = () => {
+  let options = Object.assign({}, config.defaultOptions)
+  options.cwd = path.resolve(__dirname, '../../..')
+  return options
+}
+
+const runPylint = (args, continueOnFail = false) => {
+  let cmd_options = getDefaultOptions()
+  cmd_options.continueOnFail = continueOnFail
+  if (process.platform !== 'win32') {
+    util.run('which', ['pylint'], cmd_options)
+  } else {
+    util.run('where', ['pylint'], cmd_options)
+  }
```

Comment:
I'm guessing this is so we throw an error if pylint isn't in the environment's path? If so, should we wrap with a `try ... catch` and throw a meaningful error like `'pylint not found in path'`. Otherwise I think it may just print `null` to the terminal.

Reply:
Addressed in https://github.com/brave/brave-core/pull/8782/files#diff-51c6e3f7ee7e15fa6a0611b0c64b87a7ca9a800564bf30e0a9107326d3ef6039R35 by checking the return status.